test/test.mk

```
@@ -16,11 +16,15 @@ mode-host   := -t $(or $(TOPOLOGY),/etc/infamy.dot)
 mode-run    := -t $(BINARIES_DIR)/qemu.dot
 mode        := $(mode-$(TEST_MODE))
 
-
 binaries-$(ARCH) := $(addprefix $(IMAGE)-$(ARCH),.img -disk.img .pkg)
 binaries-x86_64  += OVMF.fd
 binaries := $(foreach bin,$(binaries-$(ARCH)),-f $(BINARIES_DIR)/$(bin))
 
+# Common transport override for minimal defconfigs
+ifneq ($(BR2_PACKAGE_ROUSETTE),y)
+export INFAMY_TRANSPORT := netconf
```

```suggestion
export INFAMY_ARGS := --transport netconf
```
I think we can already pass infamy arguments via this variable. This is how `test/env` passes the physical topology path to the tests, for example.

---

Sure, works with an extra `--env INFAMY_ARGS="$INFAMY_ARGS"` to docker wrapper.

I tried almost this first, but using `INFAMY_EXTRA_ARGS` which blew` up on me and I never got working with `make test-sh`.